src/secp256k1/secp256k1.c

```diff
@@ -216,36 +216,20 @@ void secp256k1_scratch_space_destroy(const secp256k1_context *ctx, secp256k1_scr
 }
 
 static int secp256k1_pubkey_load(const secp256k1_context* ctx, secp256k1_ge* ge, const secp256k1_pubkey* pubkey) {
-    if (sizeof(secp256k1_ge_storage) == 64) {
-        /* When the secp256k1_ge_storage type is exactly 64 byte, use its
-         * representation inside secp256k1_pubkey, as conversion is very fast.
-         * Note that secp256k1_pubkey_save must use the same representation. */
-        secp256k1_ge_storage s;
-        memcpy(&s, &pubkey->data[0], sizeof(s));
-        secp256k1_ge_from_storage(ge, &s);
-    } else {
-        /* Otherwise, fall back to 32-byte big endian for X and Y. */
-        secp256k1_fe x, y;
-        secp256k1_fe_set_b32(&x, pubkey->data);
-        secp256k1_fe_set_b32(&y, pubkey->data + 32);
-        secp256k1_ge_set_xy(ge, &x, &y);
-    }
+    /* When the secp256k1_ge_storage type is exactly 64 byte, use its
+     * representation inside secp256k1_pubkey, as conversion is very fast.
+     * Note that secp256k1_pubkey_save must use the same representation. */
+    secp256k1_ge_storage s;
+    memcpy(&s, &pubkey->data[0], sizeof(s));
+    secp256k1_ge_from_storage(ge, &s);
```

Comment:
I think this should be reverted, and warning should be ignored in this place.
sizeof(secp256k1_ge_storage) == 64 is always true in some OSes, but on other OSes will fail.